torchdata/dataloader2/reading_service.py

```diff
@@ -261,7 +277,7 @@ def initialize(self, datapipe: DataPipe) -> DataPipe:
             )
             (process, req_queue, res_queue) = communication.eventloop.CreateProcessForDataPipeline(
                 ctx,
-                datapipe,
+                replicable_dp,
```

Comment:
To confirm my understanding, after this change process.start will only execute the replicable DataPipe, then we don't have the initialize_iteration ordering issue in SequentialRS?

Reply:
Order still matters especially when we need to let the first RS to specify which part should be sent to dispatching process.